roles/tarzan/tasks/main.yml

```diff
@@ -16,9 +16,20 @@
               backup=no
   tags: tarzan
 
+- name: add the lua_path to funk_004's environment via the sourceme_upps script
+  lineinfile: dest={{ ngi_pipeline_conf }}/{{ bash_env_upps_script }}
+              line='export LUA_PATH={{ lua_path }}'
+              backup=no
```

Comment:
Ah! OK. Yeah, that is true. I understand what you mean. 

It would be a bit more "consistent" if we created these scripts by only using e.g. the 'lineinfile` module, but that would make it a bit more messy if we want to have static content in the files as well. 

In this case I guess it is mostly the comment lines in the sourcme scripts. We would then have to create a `lineinfile` task for each of these comments, and also having two tasks for the site scripts that does the `source` of the common script. 

One of the drawbacks of having a mix of `lineinfile` and templates is that the `ngi_pipeline` role needs to run before the `tarzan` role, as otherwise `tarzan` would create the Uppsala script, and then if someone runs the `ngi_pipeline` role afterwards then the script will be overwritten by the template. 

Same thing applies for the `piper` role, because that adds the PATHs via `lineinfile` as well: if `piper` is run without `ngi_pipeline` having been run beforehand, and then someone decides to run `ngi_pipeline` afterwards, then the sourcme script will be overwritten. 

One can add a `meta/main.yml` for the `tarzan` and `piper` roles though, that says that `ngi_pipeline` is a dependency. That will cause the `ngi_pipeline` to always run before running the role in question. We should perhaps add this.